+ source /root/mugen-riscv/libs/locallibs/common_lib.sh
++ python3 --version
++ '[' 0 -eq 0 ']'
++ source /root/mugen-riscv/libs/locallibs/common_lib_python.sh
+ main
++ type -t post_test
+ '[' -n '' ']'
+ trap POST_TEST_DEFAULT EXIT INT HUP TERM
+ rpm -qa
+ grep expect
++ type -t config_params
+ '[' -n '' ']'
++ type -t pre_test
+ '[' -n '' ']'
++ type -t run_test
+ '[' -n function ']'
+ run_test
+ LOG_INFO 'Start to run test.'
+ message='Start to run test.'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level info --message 'Start to run test.'
Thu Dec 29 11:49:08 2022 - INFO  - Start to run test.
+ who
++ users
++ awk '{print$1}'
+ grep
Usage: grep [OPTION]... PATTERNS [FILE]...
Try 'grep --help' for more information.
+ CHECK_RESULT 2
+ actual_result=2
+ expect_result=0
+ mode=0
+ error_log=
+ '[' -z 2 ']'
+ '[' 0 -eq 0 ']'
+ test 2x '!=' 0x
+ test -n ''
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_who.sh line 24'
+ message='oe_test_who.sh line 24'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'oe_test_who.sh line 24'
Thu Dec 29 11:49:09 2022 - ERROR - oe_test_who.sh line 24
+ return 0
+ who -b
+ grep 'system boot'
         system boot  Dec 29 11:48
+ CHECK_RESULT 0
+ actual_result=0
+ expect_result=0
+ mode=0
+ error_log=
+ '[' -z 0 ']'
+ '[' 0 -eq 0 ']'
+ test 0x '!=' 0x
+ return 0
+ who -s
+ grep pts
+ CHECK_RESULT 1
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log=
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n ''
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_who.sh line 30'
+ message='oe_test_who.sh line 30'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'oe_test_who.sh line 30'
Thu Dec 29 11:49:09 2022 - ERROR - oe_test_who.sh line 30
+ return 0
+ who --help
+ grep Usage
Usage: who [OPTION]... [ FILE | ARG1 ARG2 ]
+ CHECK_RESULT 0
+ actual_result=0
+ expect_result=0
+ mode=0
+ error_log=
+ '[' -z 0 ']'
+ '[' 0 -eq 0 ']'
+ test 0x '!=' 0x
+ return 0
+ LOG_INFO 'End to run test.'
+ message='End to run test.'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level info --message 'End to run test.'
Thu Dec 29 11:49:10 2022 - INFO  - End to run test.
+ CASE_RESULT 0
+ case_re=0
+ test -z 2
+ test 2 -gt 0
+ LOG_ERROR 'failed to execute the case.'
+ message='failed to execute the case.'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'failed to execute the case.'
Thu Dec 29 11:49:11 2022 - ERROR - failed to execute the case.
+ exit 2
+ POST_TEST_DEFAULT
+ LOG_INFO 'oe_test_who.sh post_test'
+ message='oe_test_who.sh post_test'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level info --message 'oe_test_who.sh post_test'
Thu Dec 29 11:49:11 2022 - INFO  - oe_test_who.sh post_test
